skills/geoparquet/Skill.md

```
@@ -0,0 +1,344 @@
+# Public S3 files (no auth needed)
+gpio inspect s3://bucket/public-file.parquet
+
+# Private S3 files (uses ~/.aws/credentials)
+gpio inspect s3://bucket/private-file.parquet --profile my-profile
+
+# HTTP/HTTPS URLs
+gpio inspect https://example.com/data.parquet
+```
+
+For S3 writes, ensure AWS credentials are configured:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)